projects/ngx-datatable/src/lib/components/body/body-row-wrapper.component.ts

```diff
@@ -184,22 +208,26 @@ export class DataTableRowWrapperComponent<TRow = any> implements DoCheck, OnInit
     }
   }
 
-  @HostListener('contextmenu', ['$event'])
+  @HostListener("contextmenu", ["$event"])
```

Comment:
any reason why we want to switch to double quotes?

Reply:
We added prettier now and I rebased on all the new common format.